src/data_io.py

```diff
@@ -9,6 +9,8 @@
 from pydantic import BaseModel
 from tqdm import tqdm
 
+from configs.config import CONFIG
```

Comment:
ya ya put this in tha package if you're going to import it

Reply:
Wait sorry I was testing it 😅 i'm trying to leave config exclusively out of the package. it should only go in as a set of parameters during the runtime. any recommendations on how i should structure the config folder?

Reply:
I would have stuff to load configs in the package, maybe some demo configs in the repo, but otherwise user provided configs are out of the repo in some user directory. Goal should be that people dont need the repo to use the tool, e.g. "should be able to `pip install cala`" means we cant rely on repo directory structure.

Reply:
Pretty good guidelines: https://clig.dev/#configuration